common/display/virtualpanoramadisplay.cpp

```diff
@@ -492,6 +522,24 @@ int VirtualPanoramaDisplay::GetDisplayPipe() {
   return -1;
 }
 
+#ifdef HYPER_DMABUF_SHARING
+bool VirtualPanoramaDisplay::SetHyperDmaBufMode(uint32_t mode) {
+  if (hyper_dmabuf_mode_ != mode) {
+    hyper_dmabuf_mode_ = mode;
+    if (!hyper_dmabuf_mode_) {
```

Comment:
Is this expected? if so need more explain/comment here.

Reply:
Yes, this is expected. And just revised the code comments with more details. Thanks!